cli/paych.go

```diff
@@ -105,52 +107,86 @@ var paychStatusCmd = &cli.Command{
 		}
 		defer closer()
 
-		avail, err := api.PaychAvailableFunds(from, to)
+		avail, err := api.PaychAvailableFundsByFromTo(from, to)
 		if err != nil {
 			return err
 		}
 
-		if avail.Channel == nil {
```

Comment:
The logic of formatting the channel status output is the same, it's just been extracted into a method that's called by both `paych status` and `paych status-by-from-to`